docs/Deploying-Pega-on-AKS.md

```diff
@@ -521,6 +521,18 @@ helm inspect values pega/pega > <local filepath>/aks-demo/pega.yaml
 example:
    `connectionProperties: "selectMethod=cursor;Authentication=ActiveDirectoryMSI;msiClientId=<clientId>;"`
 
+    As an authentication alternative, you can configure an AAD Pod Identity to manage authentication access for VMSS of your cluster to Azure SQL database. use 
```

Comment:
Fix context (this is pretty complex stuff) and then maybe confirm that it is clear how you add the `aadpodidbinding: <label>` label: there are no quotes here but don't all of the parameters require text to be parsed using double quotes?

As an authentication alternative, you can configure an AAD Pod Identity to manage authentication access for VMSS of your cluster to Azure SQL database. Use the "web" `tier.podLabels` parameter to pass a `aadpodidbinding` label to your pod to enable it to discover your AAD Pod Identity.